### PR TITLE
[RUN-3233] Fix FontFallbackMap Asserts again

### PR DESCRIPTION
**Pull request by Domiii**
* https://linear.app/replay/issue/RUN-3109/order-fontfallbackmaplock-#comment-0c6af3fb

* Build: https://buildkite.com/replay/chromium-build/builds/3323

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -155,7 +155,7 @@ const autoninja =
   currentPlatform() == "windows" ? "autoninja.bat" : "autoninja";
 // make the windows build verbose so we can see what's going on
 const platformAutoNinjaArgs = currentPlatform() == "windows" ? ["-v"] : [];
-spawnChecked(autoninja, [...platformAutoNinjaArgs, "-C", outdir, "chrome"], {
+spawnChecked(autoninja, ["-v", ...platformAutoNinjaArgs, "-C", outdir, "chrome"], {
```
Comment (klochek) [inline build.js:158]: ?

Comment (Domiii) [reply build.js:158]: That was an attempt at [RUN-3228](https://linear.app/replay/issue/RUN-3228/compiler-errors-dont-show-in-ci), but it did not work

Comment (klochek) [reply build.js:158]: Sorry: is this supposed to be in this PR?  How does it relate?  If it does not work, why is it here?

Comment (Domiii) [reply build.js:158]: I removed it, but... the push didn't work 😠... Have to do it tmrw...

Comment (Domiii) [reply build.js:158]: Why, you little...
image

**Review by klochek — APPROVED** [review]

**Pull request merged**